app/views/users/index.html.erb

```diff
@@ -0,0 +1,15 @@
+
+<h1>Users</h1>
+<div class="user-container">
+<% @users.each do |user| %>
+  <div class="user-card">
+    <div class="user-image">
+          <img src="<%= user.photo_link %>" alt="<%= user.name %> photo" class="img-fluid rounded-circle">
+      </div>
+    <div class="user-info">
+      <h3><%= link_to user.name, user_path(user) %></h3>
```

Comment:
- [ ] Asides fixing the user index page, you need to click your links and ensure they work. When I click the user name's link I am taken to this error page. To fix this, pay attention to the routing comment below

  ![image](https://github.com/hydaline-code/myblogapp/assets/36542986/bc581a04-9f50-45ff-bc83-5fb7c20076c8)